src/modules/notifications/notifications.controller.ts

```diff
@@ -39,16 +41,33 @@ export class NotificationsController {
     const userId = user.id;
     return this.notificationsService.markNotificationAsRead(markNotificationAsRead, notification_id, userId);
   }
-  @Delete('/clear')
-  @ApiOkResponse({ type: MarkAllNotificationAsReadResponse, description: 'Notifications cleared successfully.' })
-  @ApiUnauthorizedResponse({ type: MarkAllNotificationAsReadError, description: 'Unauthorized' })
-  @ApiInternalServerErrorResponse({ type: MarkAllNotificationAsReadError, description: 'Internal Server Error' })
-  @ApiOperation({ summary: 'Marks all notifications a read' })
-  async markAllNotificationsAsRead(@Req() request: Request) {
-    const user = request['user'];
-
-    const userId = user.id;
 
-    return this.notificationsService.markAllNotificationsAsReadForUser(userId);
```

Comment:
This is someone else's code you have taken off. Please add it back.

```suggestion
 @Delete('/clear')
  @ApiOkResponse({ type: MarkAllNotificationAsReadResponse, description: 'Notifications cleared successfully.' })
  @ApiUnauthorizedResponse({ type: MarkAllNotificationAsReadError, description: 'Unauthorized' })
  @ApiInternalServerErrorResponse({ type: MarkAllNotificationAsReadError, description: 'Internal Server Error' })
  @ApiOperation({ summary: 'Marks all notifications a read' })
  async markAllNotificationsAsRead(@Req() request: Request) {
    const user = request['user'];

    const userId = user.id;

    return this.notificationsService.markAllNotificationsAsReadForUser(userId);
```

Reply:
Okay